developer_guide/installation/installation_archive.rst

```diff
@@ -0,0 +1,123 @@
+Installing Akeneo PIM with the Archive
+======================================
+
+This document provides step by step instructions to install the PIM with the archive on an environment that fulfill the expected :doc:`system_requirements/system_requirements`.
+
+Getting Akeneo PIM
+------------------
+
+Downloading the archive
+***********************
+
+From http://www.akeneo.com/download you can choose to download a PIM edition with (called *icecat*) or without (called *minimal*) demo data. If you prefer, you can also download them directly from the command line:
```

Comment:
I would have changed the formulation here:
`From http://www.akeneo.com/download you can choose to download a PIM edition with (called *icecat*) or without (called *minimal*) demo data`
for:
`You can choose to download a PIM edition along with demo data (called *icecat*) or not (called *minimal*) from the download page http://www.akeneo.com/download`.
This way when we read the with/without we already know we are talking about fixtures.